<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelbienvenue">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>67</y>
      <width>717</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vous pouvez écrire ou coller un texte.</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelUrlOuCheminFichier">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenue au détecteur de langues. Collez un chemin de fichier ou un lien vers une page web.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>http://www.legorafi.fr/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonParcourir">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>30</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Parcourir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonURL">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Page Web</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBoxLancerDetecteur">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>176</width>
      <height>27</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelReponse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditPrTexte">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>531</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
